### PR TITLE
Added tests for Randoms Spreads

### PR DESCRIPTION
**Pull request by bijan2007**
Refactored code

### CODE REVIEW

File: nightwatch.json [hunk 1]
```diff
@@ -130,7 +130,7 @@
          }
       },
       "browserstack":{
-         "launch_url":"",
+         "launch_url":"https://www.binaryqa09.com",
```
Comment (uchediwe) [inline nightwatch.json]: update to empty string

File: nightwatch.json [hunk 2]
```diff
@@ -146,7 +147,7 @@
             "acceptSslCerts":true,
             "browserstack.user":"ucheonwudiwe1",
             "browserstack.key":"xs7wt7YzLMc3scfASywx",
-            "browserstack.local":"false",
+            "browserstack.local":"true",
```
Comment (uchediwe) [inline nightwatch.json]: I don't quite recall, does this prevent running on browserstack from travis?  If so, let's set it back to false.

Comment (bijan2007) [inline nightwatch.json]: Done.

File: nightwatch.json (hunk 1 above)
Comment (bijan2007) [inline nightwatch.json]: Done.

**Pull request merged**